lib/task1.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
What the code is inside the `lib` folder? I see that it's almost similar to your solution.

Reply:
It's code version after launching babel.